dotcom-rendering/src/types/front.ts

```diff
@@ -503,7 +503,31 @@ type FESeoDataType = {
 
 type FEFrontPropertiesType = {
 	isImageDisplayed: boolean;
-	commercial: Record<string, unknown>;
+	commercial: {
```

Comment:
We are sending [commercialProperties](https://github.com/guardian/frontend/blob/main/common/app/common/commercial/CommercialProperties.scala#L19-L22) already from frontend, we just need to explicitly say this in the JSON schema we are expecting.

Reply:
for example:

![image](https://github.com/guardian/dotcom-rendering/assets/19683595/09e2c11b-e881-4461-b7b5-115b415d9c49)